src/main/java/org/czekalski/petapiintegration/controllers/AuthorizationController.java

```diff
@@ -1,61 +1,160 @@
 package org.czekalski.petapiintegration.controllers;
 
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.czekalski.petapiintegration.apiClient.v1.dto.AnimalDto;
+import org.czekalski.petapiintegration.apiClient.v1.dto.AnimalsListDto;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.security.oauth2.client.OAuth2RestTemplate;
-import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.io.IOException;
+
 @RestController
 public class AuthorizationController {
 
-
-
     @Value("${messages.base-uri}")
     private String messagesBaseUri;
```

Comment:
To także powinno być ustawiane w konstruktorze

Reply:
Dobra tutaj to nie do końca rozumiem bo to jest w sumie String tylko i on służy później jako parametr, ale na przyszłość to zmienić na coś takiego?

`  private final String messagesBaseUri;

    private final OAuth2RestTemplate auth2RestTemplatee;

    public AuthorizationController(@Value("${messages.base-uri}")String messagesBaseUri, OAuth2RestTemplate auth2RestTemplatee) {
        this.messagesBaseUri = messagesBaseUri;
        this.auth2RestTemplatee = auth2RestTemplatee;
    }`